src/main/kotlin/no/nav/helsearbeidsgiver/dokarkiv/DokArkivClient.kt

```diff
@@ -46,6 +47,8 @@ class DokArkivClient(
         /** Unik id for forsendelsen som kan brukes til sporing gjennom verdikjeden. */
         eksternReferanseId: String,
         callId: String,
+        /** Hvilken mottakskanal dokumentet er sendt inn gjennom feks NAV_NO for skjemaer på nav.no **/
+        kanal: Kanal = Kanal.NAV_NO,
```

Comment:
Tror jeg foretrekker breaking change istedenfor her. Kjipt om vi glemmer oss og plutselig journalfører med feil kanal. Klienten er vel uansett på versjon 0, så null stress å breake med minor-bump imo.

Reply:
Helt greit for meg. Da fjerner jeg default og bumper til 0.5.0